### DISCUSSION TITLE
Add support for AMD Radeon 890M GPU

### DISCUSSION THREAD

**Discussion by zydjohnHotmail** [dotnet/Silk.NET#2401] [Q&A] [answered]
# Summary of feature
I want to know if the nuget package or at least for version 3.0 can support AMD Radeon 890M GPU.
# Comments
Hi, as many people are using latest CPU from AMD, esp. for those AMD Ryzen AI 9 HX 370 CPU with built-in Radeon 890M GPU, I want to know if the repo will support this new GPU, or at least for the new version 3.0.

# Does this have a proposal?
Check the documentation/proposals folder. If it doesn't have one, you may need to create one if you're making **massive breaking changes**.

**Comment by aquagoose** [comment, marked as answer]
Whether or not a specific GPU is supported is not up to Silk.NET but the GPU driver. The library has no bearing on what GPUs are or aren't supported, as it just provides bindings to the APIs directly. Although considering how recently the GPU was released I'm willing to bet all the modern graphics APIs will work fine on the GPU.
